### ISSUE TITLE
Use decentralized storage system for images such as IPFS or Swarm

### ISSUE THREAD

**Issue by layters** [layters/testshop#115]
```
Details
```
TL;DR: Basically, I need to figure out how I would store images in the DHT, now that that the local database is no longer a viable option.

To-do: Use a decentralized storage system designed for large files (images), such as IPFS or Swarm instead of storing large files such as images in a DHT, which can put a strain on the network, as it requires multiple nodes to store and retrieve the file. This can lead to slower performance and higher bandwidth usage, especially if many users are uploading and downloading large avatar images.

Edit: Unfortunately, this requires running an IPFS node. I could use a different storage method that does not require running any sort of node.

```
References
```
https://github.com/vasild/cpp-ipfs-http-client
